core/message-tags-3.3.md

```diff
@@ -80,6 +88,36 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### The `TAGMSG` tag-only event
+
+A new event `TAGMSG` is defined for sending messages with tags but no text content.
+This event MUST be delivered to targets in the same way as `PRIVMSG` and `NOTICE`
+events, taking into account channel membership and modes.
+
+Servers MAY apply moderation to this event using existing or newly specified modes or
+configuration.
+
+These events MUST NOT be delivered to clients that haven't negotiated the message tags
+capability and MAY be rejected if no tags are included.
+
+If this event is sent without any tags, servers SHOULD reject it with a standard
+`ERR_NEEDMOREPARAMS` error numeric.
+
+See `PRIVMSG` for more details on replies and examples.
```

Comment:
I mean, that sentence is taken from the end of NOTICE. Should I link the word `PRIVMSG` to https://tools.ietf.org/html/rfc1459#section-4.4.1 ?